### PR TITLE
Tidy up code

### PR DESCRIPTION
**Pull request by bryanyee33**
Removes unnecessary print statement.
Removes repetitive requireNonNull() checks.

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -135,9 +133,10 @@ public boolean hasAsset(Asset asset) {
      */
     public void editAsset(Asset target, Asset editedAsset) {
         requireNonNull(target);
+        requireNonNull(editedAsset);
```
Comment (aureliony) [inline src/main/java/seedu/address/model/AddressBook.java:136]: Would it be better to shorten this to 1 line using the varargs signature `requireAllNonNull(Object...)`?

**Review by aureliony — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (bryanyee33) [reply src/main/java/seedu/address/model/AddressBook.java]: Moved it to Assets#edit(), using the 1 line requireAllNonNull().

**Review by aureliony — APPROVED** [review]
LGTM
